client/src/pages/track/Track.svelte

```diff
@@ -1,114 +1,145 @@
+            <Button>
+                <Notification /> Subscribe to Push Notifications
+            </Button>
+            <section>
+                <table>
+                    <tr>
+                        <td><p class="header-color"><b>Overview</b></p></td>
+                        <td></td>
+                    </tr>
+                    <tr>
+                        <td><b>Document Title</b></td>
+                        <td>{overview.title}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Document Tracking Number</b></td>
+                        <td>{trackingNumber}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Document Category</b></td>
+                        <td>{overview.category}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Document For</b></td>
+                        <td>{overview.documentFor}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Document Remarks</b></td>
+                        <td>{overview.documentRemark}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Originating Office</b></td>
+                        <td>{overview.origin}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Current Office</b></td>
+                        <td>{overview.current}</td>
+                    </tr>
+                    <tr>
+                        <td><b>Document Status</b></td>
+                        <td>{overview.status}</td>
+                </table>
+                <br>
+                <table>
+                    <td><p class="header-color"><b>File Attachment</b></p></td>
+                    <tr>No file attachment.</tr>
+                </table>
+                <br>
+                <table>
+                    <td><p class="header-color"><b>Paper Trail</b></p></td>
+                    <tr>
+                        <td><b>Office</b></td>
+                        <td><b>Creation</b></td>
+                        <td><b>Action</b></td>
+                        <td><b>Remarks</b></td>
+                        <td><b>Evaluator</b></td>
+                    </tr>
+                    {#each trail as { target, creation, status, remark, email }}
+                        <tr>
+                            <td>{target}</td>
+                            <td>{creation}</td>
+                            <td>{status}</td>
+                            <td>{remark}</td>
+                            <td>{email}</td>
+                        </tr>
                     {/each}
-                </tr>
-            {/each}
-        </table>
-    </section>
+            </section>
+        {/if}
+    {:catch error}
+        <h1>Uh oh!</h1>
+        <p>Something went wrong. Kindly re-check your tracking id above.</p>
+        <p style="color: red;">Error: {error}</p>
```

Comment:
See the [`style:property` element directive](https://svelte.dev/docs#template-syntax-element-directives-style-property).
```suggestion
        <p style:color="red">Error: {error}</p>
```